tests/pci_passthrough/test_pci_passthrough.py

```diff
@@ -0,0 +1,82 @@
+import pytest
+import logging
+
+# Requirements:
+# - an XCP-ng host (--hosts) >= 8.3 with a PGPU to hide from dom0
+
+XEN_CMDLINE = '/opt/xensource/libexec/xen-cmdline'
+
+@pytest.mark.reboot # reboots the host
+@pytest.mark.incremental # tests depend on each other. If one test fails, don't execute the others
```

Comment:
Incremental tests are not a very good idea. I regret I ever used it in the first place, because people often get bitten when they try to isolate a test to reproduce an issue. Here, we could avoid it by simply grouping "disable" and "enable" tests together so that the host gets back to its initial state when the test ends. Actually, that's already what you do in `test_access_status_manual_modification`, so I strongly suggest we do it for the rest too.

Reply:
I see what you mean, I have no preference, the tests would be longer and less specific but this doesn"t bother that much. :+1: